spiped/main.c

```diff
@@ -27,6 +28,12 @@ usage(void)
 	exit(1);
 }
 
+static void
+signal_handler(int signo)
+{
+	exit(0);
```

Comment:
This needs to be _exit since the 'exit' function can't be used from inside a signal handler: http://pubs.opengroup.org/onlinepubs/9699919799/